_sass/_uswds-theme-overrides.scss

```diff
@@ -179,15 +169,46 @@ $theme-link-reverse-active-color: "white";
 // ==========================================================================
 // Set to `false` when developing.
 $theme-show-notifications: false;
-
+$theme-focus-offset: 2px;
 
 // ==========================================================================
 // Typography - from `_uswds-theme-typography.scss`.
 // ==========================================================================
+$theme-font-type-sans: "source-sans-pro";
 $theme-global-link-styles: true;
 
 
 // ==========================================================================
 // Component Overrides - from `_uswds-theme-components.scss`.
 // ==========================================================================
+//
+// Components come with settings that allow you to easily customize globally.
+// If you need more refined styles per section of your site use custom styles.
+// You can view all of the component settings at https://designsystem.digital.gov/documentation/settings/
+// Click on `type` to get a reference of available options.
+//
+
+//
+// Buttons
+// ==========================================================================
+// https://designsystem.digital.gov/design-tokens/spacing-units/
+$theme-button-border-radius: "sm";
+
+//
+// Identifier
+// ==========================================================================
+
+//
+// Learn more about colors:
+// https://designsystem.digital.gov/design-tokens/color/system-tokens/
+//
+// Uses https://civicactions.github.io/uswds-color-tool/ to find the closest color to `$c-dhs-blue`
 $theme-identifier-background-color: "blue-cool-60v";
+
+//
+// Cards
+// ==========================================================================
+$theme-card-border-radius: "md";
+// Closest USWDS Color to $c-dhs-dark-gray
+$theme-card-border-color: "gray-60";
+$theme-card-border-width: 1px;
```

Comment:
3. These new overrides refine the USWDS components. The rest of these settings to the following:

- Set `source sans pro` as the default sans font
- Add more spacing to the focus offset border because it was bunching up too much on smaller screens
- Make the Card and Button components more square
- Reduces the width of the Card border as well as changes color